README.md

```diff
@@ -0,0 +1,159 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1>Morse Decoder Script</h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 📖 Morse Decoder <a name="about-project"></a>
+
+A simple Morse Decoder Ruby Script.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Morse Decoder**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Node.js
+
+### Setup
+
+```
+  RUN mkdir my-folder
+  RUN cd my-folder
+  RUN git clone https://github.com/fezzopro/morse-code-decoder.git
+  RUN cd morse-code-decoder
+
+```
+
+### Install
+
+
+```
+  RUN bundle install
+
+```
+
+### Usage
+
+ - Install Ruby to your environment
+ - **RUN** `ruby morse_code_decoder.rb`
+
+### Run tests
+
+ - No Tests yet
+
+### Deployment
+
+ - No deployment yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **Felix KAMANA**
+
+- GitHub: [@githubhandle](https://github.com/fezzopro)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/felix-kamana/)
+
```

Comment:
![image](https://github.com/fezzopro/morse-code-decoder/assets/93039359/be765513-daa2-4cea-b59c-818870dcbf60)

- [ ] It would be nice if you fix your LinkedIn link to make your readme descriptive and professional 👍